### PR TITLE
Fix lowercase schema name in ddl mounter

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

QA found a bug that CDC couldn't replicate `CREATE TABLE` when the schema name is not all lowercase.
The root cause is TiDB stores the `CREATE TABLE` DDL history with a lowercase schema name.

### What is changed and how it works?

Find the correct schema name from the `SchemaStorage` when we mount a DDL

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cdc/entry/mounter.go [hunk 1]
```diff
@@ -207,6 +207,14 @@ func fetchHandleValue(tableInfo *schema.TableInfo, recordID int64) (pkCoID int64
 
 func (m *Mounter) mountDDL(jobEntry *ddlJobKVEntry) (*model.DDL, error) {
 	databaseName := jobEntry.Job.SchemaName
+	// schema name in raw ddl job entry is always lowercase, try to find the
+	// correct value from schema storage
+	if m.schemaStorage != nil {
```
Comment (leoppro) [inline cdc/entry/mounter.go]: we can find origin schema name in `job.BinlogInfo.DBInfo.Name.O`

Comment (amyangfei) [reply cdc/entry/mounter.go]: I'm not sure, but from TiDB DDL history the `job.BinlogInfo.DBInfo` is nil

Screen Shot 2020-02-27 at 19 11 21

**Comment by leoppro** [comment]
LGTM

**Pull request merged**